src/ads1115.toit

```diff
@@ -1,152 +1,170 @@
+  static MUX_DIFF_1_3_ ::= 0x2000  // Differential P  =  AIN1, N  =  AIN3.
+  static MUX_DIFF_2_3_ ::= 0x3000  // Differential P  =  AIN2, N  =  AIN3.
+
+  static MUX_SINGLE_0_ ::= 0x4000  // Single-ended AIN0.
+  static MUX_SINGLE_1_ ::= 0x5000  // Single-ended AIN1.
+  static MUX_SINGLE_2_ ::= 0x6000  // Single-ended AIN2.
+  static MUX_SINGLE_3_ ::= 0x7000  // Single-ended AIN3.
+
+  static SINGLE_ENDED_ ::= [MUX_SINGLE_0_, MUX_SINGLE_1_, MUX_SINGLE_2_, MUX_SINGLE_3_]
+
+
+  static PGA_MASK_ ::=   0x0E00
+  static PGA_6_144V_ ::= 0x0000  // +/-6.144V range  =  Gain 2/3.
+  static PGA_4_096V_ ::= 0x0200  // +/-4.096V range  =  Gain 1.
+  static PGA_2_048V_ ::= 0x0400  // +/-2.048V range  =  Gain 2 (default).
+  static PGA_1_024V_ ::= 0x0600  // +/-1.024V range  =  Gain 4.
+  static PGA_0_512V_ ::= 0x0800  // +/-0.512V range  =  Gain 8.
+  static PGA_0_256V_ ::= 0x0A00  // +/-0.256V range  =  Gain 16.
+
+  static MODE_MASK_ ::=   0x0100
+  static MODE_CONTIN_ ::= 0x0000  // Continuous conversion mode.
+  static MODE_SINGLE_ ::= 0x0100  // Power-down single-shot mode (default).
+
+  static DR_MASK_ ::= 0x00E0     // Values ADS1015/ADS1115.
+  static DR_128SPS_ ::= 0x0000   // 128 /8 samples per second.
+  static DR_250SPS_ ::= 0x0020   // 250 /16 samples per second.
+  static DR_490SPS_ ::= 0x0040   // 490 /32 samples per second.
+  static DR_920SPS_ ::= 0x0060   // 920 /64 samples per second.
+  static DR_1600SPS_ ::= 0x0080  // 1600/128 samples per second (default).
+  static DR_2400SPS_ ::= 0x00A0  // 2400/250 samples per second.
+  static DR_3300SPS_ ::= 0x00C0  // 3300/475 samples per second.
+  static DR_860SPS_ ::= 0x00E0  // -   /860 samples per Second.
+
+  static CMODE_MASK_ ::= 0x0010
+  static CMODE_TRAD_ ::= 0x0000  // Traditional comparator with hysteresis (default).
+  static CMODE_WINDOW_ ::= 0x0010  // Window comparator.
+
+  static CPOL_MASK_ ::= 0x0008
+  static CPOL_ACTVLOW_ ::= 0x0000  // ALERT/RDY pin is low when active (default).
+  static CPOL_ACTVHI_ ::= 0x0008  // ALERT/RDY pin is high when active.
+
+  static CLAT_MASK_ ::= 0x0004  // Determines if ALERT/RDY pin latches once asserted.
+  static CLAT_NONLAT_ ::= 0x0000  // Non-latching comparator (default).
+  static CLAT_LATCH_ ::= 0x0004  // Latching comparator.
+
+  static CQUE_MASK_ ::= 0x0003
+  static CQUE_1CONV_ ::= 0x0000  // Assert ALERT/RDY after one conversions.
+  static CQUE_2CONV_ ::= 0x0001  // Assert ALERT/RDY after two conversions.
+  static CQUE_4CONV_ ::= 0x0002  // Assert ALERT/RDY after four conversions.
+  // Disable the comparator and put ALERT/RDY in high state (default).
+  static CQUE_NONE_ ::= 0x0003
+
+
+  static GAINS_PGA_6_144V_ ::= 0
+  static GAINS_PGA_4_096V_ ::= 1  // 1x.
+  static GAINS_PGA_2_048V_ ::= 2  // 2x.
+  static GAINS_PGA_1_024V_ ::= 3  // 4x.
+  static GAINS_PGA_0_512V_ ::= 4  // 8x.
+  static GAINS_PGA_0_256V_ ::= 5  // 16x.
+
+
+  static RATES_DR_8SPS_   ::= 0 // 8 samples per second.
```

Comment:
Changed the names of these constants: the ADS1115 doesn't have the higher sample rates (that were referenced in the old code).